README.md

```diff
@@ -120,41 +124,193 @@ Each flavor represents a pair of programming language and builds tool. All the u
 
 ### Core flavor
 
-Create-cycle-app come packed with 1 core flavor:
+Create-cycle-app comes pre-packaged with this core flavor:
 
 * [cycle-scripts](https://github.com/cyclejs-community/create-cycle-app/blob/master/packages/cycle-scripts)
 
-### Custom flavour
+### Using a custom flavor
 
 Custom flavors allow generating starting projects to fulfil specific needs.
-They can be published to npm, or being used locally via the create-cycle-app CLI.
+They can be published to npm or a git repo or even be referenced locally (esp. useful when you create your _own_ custom flavor - see below)
 
 #### How to use custom flavors
 
 ##### Providing your own
-When creating a project, you can inform which flavor you want to use with the `--flavor` flag:
+When creating a project, you can specify which flavor to use with the `--flavor` flag:
 
 ```sh
 $ create-cycle-app <name> --flavor <flavor>
 ```
-Some examples of how a flavor could be specified:
 
+The flavor will be installed via `npm install` and can therefore reference any flavor resource installable via `npm`
```

Comment:
not sure I get this, what do you wanted to say with "and can therefore reference any flavor resource installable via `npm`" ?